UnitTests/TraitsMock.h

```diff
@@ -12,16 +12,19 @@ namespace UnitTest
 {
 namespace Mocks
 {
-    struct Traits : DefaultTraits
+    struct NullTimeoutTraits : DefaultTraits

```

Comment:
NeverExpiringTimeoutTraits would also work :) Null could mean many things... a least to me.